src/gluonts/mx/__init__.py

```diff
@@ -13,6 +13,8 @@
 
 # !!! DO NOT MODIFY !!! (pkgutil-style namespace package)
 
+# flake8: noqa
```

Comment:
@jaheba hope you don't mind this? I would avoid over-engineering these init modules if possible

Reply:
So what we want is to re-expose the classes and functions without sphinx noticing?

Reply:
I'm not sure what is needed for sphinx here, I just put this annotation so that we don't need to have `# noqa` at every line (which btw complicates things when multi-line imports are there, which are sometimes enforced by `black`). I just want to avoid all this style checking to not be in the way :-)

Reply:
The other way to suppress the warning would be to add these things to `__all__`. However, that would have the effect that sphinx autogen would include these entities in the modules documentation -- which is probably not what we want.

So I think this is the way to go.